src/App.jsx

```diff
@@ -6,7 +6,7 @@ import Footer from './Shared/Footer.jsx';
 import Search from './Search/Search.jsx';
 
 const App = () => (
-    <div className="h-screen grid grid-flow-col">
+    <div className="h-screen max-h-screen">
```

Comment:
can you try changing the className to "h-screen max-h-screen flex flex-col min-h-full justify-between items-stretch"? If it doesn't work that's ok, we can leave it for now

Reply:
Modified, at the least it doesn't make it worse.